gjstest/public/register.js

```diff
@@ -15,17 +15,18 @@
 
 // Helper functions for registering gjstest test cases. Use it as follows:
 //
-//     /** @constructor */
-//     function MyTestFixture() {
+//     class MyTestFixture() {
 //       // This constructor will be called for each test. Do per-test setup
 //       // here.
-//       this.objectUnderTest_ = ...;
+//       constructor() {
+//         this.objectUnderTest_ = ...;
+//       }
 //     }
 //     registerTestSuite(MyTestFixture);
 //
-//     // Add a test case named ReturnsFalse to the fixture. The full name of
-//     // test, as reported in the test output, is MyTestFixture.ReturnsFalse.
-//     addTest(MyTestFixture, function ReturnsFalse() {
+//     // Add a test case named returnsFalse to the fixture. The full name of
+//     // test, as reported in the test output, is MyTestFixture.returnsFalse.
+//     addTest(MyTestFixture, function returnsFalse() {
```

Comment:
Why the change to capitalization?

Reply:
(Trying to understand if this is required by ES6 somehow.)

Reply:
My concern is that internal Google developers will reference this code and think there's some reason why UpperCamelCase function names should be used in this framework.  I realize the internal readme uses lowerCamelCase function names but I wouldn't be surprised if many devs reference this particular code example instead of the internal readme. 

This is probably not a huge concern though.  Based on this code search:
`addTest\([a-zA-Z0-9]+,\ function\ [A-Z] case:yes lang:js -f:third_party`, there appears to be only 13 files using UpperCamelCase function names.

If you think most external adopters of this codebase use UpperCamelCase method/function names, I can revert that aspect of the change.  If you don't think it really matters either way, I'd prefer to use lowerCamelCase function names.

Reply:
Sorry if I wasn't clear, but my question isn't about consistency at Google or anything, but more fundamental: what's the reason to prefer lower-case? Is there some style guide that says that's how functions should be named?